### ISSUE TITLE
Property 'children' is missing but required

### ISSUE THREAD

**Issue by JB1905** [JustFly1984/react-google-maps-api#1815]
After updating from v1.13.0 to v2.0.1 I got an error:
```
Property 'children' is missing in type '{ mapContainerStyle: { height: string; width: string; }; center: { lat: number; lng: number; }; zoom: number; }' but required in type 'Readonly<GoogleMapProps>'
```

Error ocurred for configuration:

```tsx
<LoadScript googleMapsApiKey={API_KEY}>
	<GoogleMap
		center={{ lat, lng }}
		zoom={zoom}
    />
</LoadScript>
```

Everything works for:

```tsx
<LoadScript googleMapsApiKey={API_KEY}>
	<GoogleMap
		center={{ lat, lng }}
		zoom={zoom}
    >
		Dummy content here...
	</GoogleMap>
</LoadScript>
```

The problem is connected with types in `GoogleMap.tsx`:

```tsx
export interface GoogleMapProps {
    children: React.ReactNode; // current (error)
    children?: React.ReactNode; // updated (working)
	...
```

**Comment by JustFly1984** [comment]
@JB1905  released New version: 2.0.2

**Comment by JB1905** [comment]
Thanks for quick fix!
